streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBGenericOptionsToDbOptionsColumnFamilyOptionsAdapter.java

```diff
@@ -1662,16 +1673,95 @@ public ConcurrentTaskLimiter compactionThreadLimiter() {
         return columnFamilyOptions.compactionThreadLimiter();
     }
 
+    @Override
     public Options setCompactionFilter(final AbstractCompactionFilter<? extends AbstractSlice<?>> compactionFilter) {
         columnFamilyOptions.setCompactionFilter(compactionFilter);
         return this;
     }
 
+    @Override
     public Options setCompactionFilterFactory(final AbstractCompactionFilterFactory<? extends AbstractCompactionFilter<?>> compactionFilterFactory) {
         columnFamilyOptions.setCompactionFilterFactory(compactionFilterFactory);
         return this;
     }
 
+    @Override
+    public Options setEnableBlobFiles(final boolean enableBlobFiles) {
```

Comment:
Started here, they are options for blobsDB. In the upstream `Options.java`, there are comments at the beginning and the end of the options. Could we add something similar here?
```java
 //
  // BEGIN options for blobs (integrated BlobDB)
  //
```
```java
 //
  // END options for blobs (integrated BlobDB)
  //
```
ref: https://github.com/facebook/rocksdb/blob/main/java/src/main/java/org/rocksdb/Options.java#L1996-L1998

Thanks.

Reply:
TFTR @showuon! I've added in the comments, let me know how it looks.